docs/signer.md

```diff
@@ -0,0 +1,354 @@
+  imageVulnerabilityRequirements:
+    maximumFixableSeverity: MEDIUM
+    maximumUnfixableSeverity: MEDIUM
+    allowlistCVEs:
+      - projects/goog-vulnz/notes/CVE-2020-10543
+      - projects/goog-vulnz/notes/CVE-2020-10878
+      - projects/goog-vulnz/notes/CVE-2020-14155
+```
+
+Here are the valid values for severity levels.
+
+| Value       | Outcome |
+|-------------|----------- |
+| BLOCK_ALL | Block all vulnerabilities except listed in allowlist. |
+| LOW | Allow Containers with Low  vulnerabilities. |
+| MEDIUM | Allow Containers with Low and Medium vulnerabilities. |
+| HIGH  | Allow Containers with Low, Medium & High vulnerabilities. |
+| CRITICAL  | Allow Containers with all known vulnerability levels (LOW to CRITICAL). |
+| ALLOW_ALL | Allow all vulnerabilities, including vulnerability with unknown severity levels. |
+
+### Signing
+
+Signing is the process of creating and storing an attestation for an image.
+To create an attestation, a private signing key is needed. The signer tool now
+supports three types of signing methods: PGP keys, PKIX keys, or signing via [Cloud KMS] (https://cloud.google.com/kms).
+
+By default, the signing helper will also upload the attestation occurrence to [Google Container Analysis](https://cloud.google.com/container-registry/docs/container-analysis),
+and the uploaded attestation can be used by both Binary Authorization and Kritis for enforcement-time decisions.
+
+#### Supported key types
+
+The tool currently supports three types of keys:
+
+- PGP keys using `-pgp_private_key` and optionally `-pgp_passphrase`  if the key is passphrase protected.
+- PKIX keys using `-pkix_private_key` and `-pkix_alg`. Supported PKIX algorithms match those [supported by Binary Authorization](https://cloud.google.com/sdk/gcloud/reference/container/binauthz/attestors/public-keys/add#--pkix-public-key-algorithm).
+- Cloud KMS using with `-kms_key_name` and `-kms_digest_alg`. 
+
+## Tutorial
+
+The tutorial will walk through how to run the signing helper tool in a local environment.
+
+1. Build the tool.
+
+    ```shell
+    go build -o ./signer ./cmd/kritis/signer
+    ```
+
+2. Enable GCP services.
+
+    The signing tool needs to access a number of Google Cloud Platform (GCP) services.
+First we need to pick a GCP project and enable those services within the project.
+
+    1. Set the default GCP project used by `gcloud`.
+
+        ```shell
+        export PROJECT_ID=[PROJECT ID]
+        ```
+
+    2. Set the default GCP project used by `gcloud`.
+
+        ```shell
+        gcloud config set project ${PROJECT_ID}
+        ```
+
+    3. Run `gcloud` to enable services within a project.
+
+        ```shell
+        gcloud services enable \
+          cloudbuild.googleapis.com \
+          containerregistry.googleapis.com \
+          containeranalysis.googleapis.com \
+          containerscanning.googleapis.com \
+          cloudkms.googleapis.com # If using Cloud KMS
+        ```
+
+3. Setting up GCP credentials.
```

Comment:
"Setting up GCP credentials" -> "Set up GCP credentials"

Reply:
it's more than enable roles? we also ask the user to download account and set up environment variable. WDYT?